specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/examples/ApiManagementCreateApiDiagnostic.json

```diff
@@ -0,0 +1,119 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2018-06-01-preview",
+    "subscriptionId": "subid",
+    "diagnosticId": "applicationinsights",
+    "apiId": "57d1f7558aa04f15146d9d8a",
+    "parameters": {
+      "properties": {
+        "enabled": true
```

Comment:
>"enabled": true [](start = 8, length = 15)

The CI is complaining about this property in the example [https://travis-ci.org/Azure/azure-rest-api-specs/jobs/378880669](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/378880669)

Reply:
fixed